LIFECYCLE.md

```diff
@@ -0,0 +1,16 @@
+# Lifecycle Management
+
+## Device Lifecycle scope
+
+High level requirements on a DPU/IPU or subsequently a "device"
+
+* a way to reboot a device
+  * if connected to a host, the host should be able to survive a device reset
```

Comment:
here you can link to BOOTSEQ.md

Reply:
Clearly a device that is exposing PCIe to the host cant' reset fully.
What you mean by device reset is perhaps update/reset of the OS (and not the entire device), or the part that needs patching (e.g. new datapath to upgrade xPU's network/storage/telemetry functions). It is worth clarifying what reset is sought after here should be very clear and tied to the rough components of an xPU that we can use a base for our discussion.

Reply:
sometimes the device will either reset fully or crash due to bug, we need to accommodate what happens there... 
Is PCIe DPC going to help us, for example ? Will host just crash ?

Reply:
I think this needs more discussion.  Though perhaps that discussion should be more targeted at the contents BOOTSEQ.md?  My plan is to replace this bullet with the link to the BOOTSEQ.md.

Reply:
go for it